src/main/java/de/retest/recheck/cli/subcommands/Commit.java

```diff
@@ -102,6 +128,39 @@ private void applyChanges( final Persistence<SutState> persistence, final SuiteC
 		}
 	}
 
+	private void checkForWarningMessage( final ReviewResult changeSets ) {
```

Comment:
Maybe extract both methods (`checkForWarningMessage` and `getWarningMessage`) into a separate class (be it a util or an actual object). Then you can also easily write unit tests.